patterns-base/src/main/resources/patterns.json

```diff
@@ -60,6 +60,12 @@
       "level": "Info",
       "enabled": true
     },
+    {
+      "patternId": "Custom_Scala_GetCalls",
+      "category": "ErrorProne",
+      "level": "Warning",
+      "enabled": true
```

Comment:
Do we want it enabled by default?
Given it will sometimes provide false positives

Reply:
You're right, I'll put it as disabled